### PR TITLE
Rollout custom notification ads to 100% for Nightly (Windows, Mac).

### PR DESCRIPTION
**Pull request by aseren**
Uplift of https://github.com/brave/brave-variations/pull/697

Rollout `CustomNotificationAds` to 100% for Nightly on Windows and Mac platforms with parameters:

* Windows: `normalized_display_coordinate_x/1/normalized_display_coordinate_y/0/inset_x/0/inset_y/72` 
* Mac: `normalized_display_coordinate_x/1/normalized_display_coordinate_y/0/inset_x/0/inset_y/220` 

`use_same_z_order_as_browser_window` parameter value is set to `true` by default in the source code, so the custom notification ad is using the same z-order as the browser window.

# Test plan

* Start with the fresh profile
* Join Brave rewards
* Trigger a custom notification ad
* Make sure that the custom notification ad appears at the top right corner with:
  - 72 px inset from the top on Windows
  - 220 px inset from the top on MacOS

### CODE REVIEW

**Review by tmancey — DISMISSED** [review]
LGTM++

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. 👍 QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/697#issuecomment-1692168862.

**Pull request merged**
